kiva/pdfmetrics.py

```diff
@@ -65,17 +71,17 @@ def parseAFMFile(afmFileName):
     needed for embedding.  A better parser would accept
     options for what data you wwanted, and preserve the
     order."""
-
-    lines = open(afmFileName, 'r').readlines()
+    with open(afmFileName, 'r') as f:
+        lines = [line.strip() for line in f]
```

Comment:
Hmmm. this requires a refactor no matter what. Neither `string.split` nor `string.strip` exist in python 3